environment.yml

```diff
@@ -3,4 +3,5 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - gdal
+  - python=3.8
```

Comment:
Do you need to specify python version in the conda env? We _should_ be able to work w/ multiple Python envs.

Reply:
right, i removed the python  from this